tests/test_loan_search.py

```diff
@@ -50,29 +50,32 @@ def test_search_loans_by_patron_pid(indexed_loans):
 
 def test_search_loans_by_patron_and_item(indexed_loans):
     """Test retrieve loan list by patron and items."""
-    search_result = search_by_patron_item(patron_pid="1",
-                                          item_pid="item_returned_3").execute()
+    search_result = search_by_patron_item_or_document(
+        patron_pid="1",
+        item_pid="item_returned_3").execute()
     assert search_result.hits.total == 1
 
-    search_result = search_by_patron_item(patron_pid="1",
-                                          item_pid="not_existing").execute()
+    search_result = search_by_patron_item_or_document(
+        patron_pid="1",
+        item_pid="not_existing").execute()
     assert search_result.hits.total == 0
 
-    search_result = search_by_patron_item(patron_pid="999999",
-                                          item_pid="item_returned_3").execute()
+    search_result = search_by_patron_item_or_document(
+        patron_pid="999999",
+        item_pid="item_returned_3").execute()
     assert search_result.hits.total == 0
 
 
 def test_search_loans_by_patron_and_item_filtering_states(indexed_loans):
     """Test retrieve loan list by patron and items filtering states."""
-    search = search_by_patron_item(patron_pid="1",
-                                   item_pid="item_returned_3",
-                                   filter_states=['ITEM_RETURNED'])
+    search = search_by_patron_item_or_document(patron_pid="1",
+                                               item_pid="item_returned_3",
+                                               filter_states=['ITEM_RETURNED'])
     search_result = search.execute()
     assert search_result.hits.total == 1
 
-    search = search_by_patron_item(patron_pid="1",
-                                   item_pid="item_returned_3",
-                                   filter_states=['ITEM_AT_DESK'])
+    search = search_by_patron_item_or_document(patron_pid="1",
+                                               item_pid="item_returned_3",
+                                               filter_states=['ITEM_AT_DESK'])
```

Comment:
Could you please add a test for searching by document_pid as this is the new feature?